src/resources/tls.cc

```diff
@@ -288,7 +288,7 @@ Object* tls_error(MbedTlsResourceGroup* group, Process* process, int err) {
     buffer[used + 1] = ' ';
     buffer[used + 2] = '\0';
     used += 2;
-    sprintf(buffer + used, "Cert depth %d:\n", group->error_depth());
+    snprintf(buffer + used, sizeof(buffer) - used, "Cert depth %d:\n", group->error_depth());
```

Comment:
snprintf doesn't set the trailing `\0` if the size isn't enough.
```suggestion
    snprintf(buffer + used, sizeof(buffer) - used, "Cert depth %d:\n", group->error_depth());
    buffer[sizeof(buffer) - 1] = '\0';
```